docs/dev_guide/diagnostic_ref/dpct1063.rst

```diff
@@ -8,7 +8,7 @@ Message
 
 .. _msg-1063-start:
 
-Advice parameter is device-defined. You may need to adjust it.
+Advice parameter is device-defined``<addtional action>``. You may need to adjust it.

```

Comment:
Spelling error and spacing issue: Add space after device-defined and update spelling to "additional"